integration_tests/config.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+CONTAINER="kw-integration"
+IMAGE="kw-raw"
```

Comment:
Single-quotes. Btw, do we really need this file? Can we squeeze it into the util.sh and ensure we always include this file?